adds topic description for alternatives to capitalist structures (#43)
datatogether/reading_datatogether: README.md

### Alternatives to Capitalist Structures

> When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives: reduction of privacy, control from advertisers, and concentration of data ownership into a few corporations are all justified by capitalism's imperative for delivery of maximum value to shareholders.

Reviewer: list structure year a bit hard to follow -- is everything after the colon fears // challenges? Seems like there isn't agreement between items

Author: should have been a semicolon instead of a colon I think. Yes, the list is of fears & challenges– I think there's grammatical agreement, am I missing something? Feel free to take a crack at it

> When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives: reduction of privacy, control from advertisers, and concentration of data ownership into a few corporations are all justified by capitalism's imperative for delivery of maximum value to shareholders.

Reviewer: ```suggestion
When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives; the reduction of privacy, disproportional influence by advertisers, and concentrated ownership of data by a few corporations are all seemingly justified by the capitalist imperative to deliver maximum value to shareholders.
```

Reviewer: @Frijol suggested above

Author: thanks! (Also, cool feature! I didn't know you could do that)
